### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by PyCore23**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,42 @@
...
+class Dog(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name, 7, is_hungry=is_hungry)
+
+    def bring_slippers(self) -> None:
+        print("The slippers delivered!")
+
+
+def feed_animals(animals: list) -> int:
+    return sum([animal.feed() for animal in animals])
```
Comment (Bohdan-Salamakha) [inline app/main.py:42]:
```suggestion
    return sum(animal.feed() for animal in animals)
```

File: app/main.py [hunk 2]
```diff
@@ -1 +1,42 @@
...
+
+class Dog(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name, 7, is_hungry=is_hungry)
+
+    def bring_slippers(self) -> None:
+        print("The slippers delivered!")
+
+
+def feed_animals(animals: list) -> int:
```
Comment (Bohdan-Salamakha) [inline app/main.py:41]: add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]

**Review by pavlejviki — CHANGES_REQUESTED** [review]

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]
fix flake8

**Review by pavlejviki — APPROVED** [review]
Good job!

**Pull request closed without merging**